dsl/src/commonMain/kotlin/it/unibo/collektive/path/impl/PathImpl.kt

```diff
@@ -3,5 +3,13 @@ package it.unibo.collektive.path.impl
 import it.unibo.collektive.path.Path
 
 internal data class PathImpl(private val path: List<Any?>) : Path {
+    private var hash = 0
```

Comment:
```suggestion
    private var hash by lazy { path.hashCode() }
```

Reply:
@DanySK is this a short but equivalent implementation? If not, discard this suggestion